server/src/main/java/io/littlehorse/server/auth/MTLSServerAuthorizer.java

```diff
@@ -1,29 +0,0 @@
-package io.littlehorse.server.auth;
-
-import io.grpc.Metadata;
-import io.grpc.ServerCall;
-import io.grpc.ServerCall.Listener;
-import io.grpc.ServerCallHandler;
-import io.littlehorse.common.util.CertificateUtil;
-import io.littlehorse.server.listener.MTLSConfig;
-
-public class MTLSServerAuthorizer implements ServerAuthorizer {
-
-    private final String commonName;
-
-    public MTLSServerAuthorizer(MTLSConfig mtlsConfig) {
-        try {
-            this.commonName = CertificateUtil.getCommonNameFromCertificate(mtlsConfig.getCaCertificate());
```

Comment:
This is where the problem was: this code pulls the Common Name out of the CA Certificate that gets mounted on the LH Server in `LHS_CA_CERT`, and uses that as the principal id for every request, while completely ignoring the incoming request's certificates.